Makefile

```diff
@@ -232,15 +228,10 @@ deploy: manifests cert-manager
 
 # Generate manifests e.g. CRD, RBAC etc.
 manifests:
-	cd config/operator/operator && kustomize edit set image controller=$(IMAGE_REF)
-	cd config/manager/manager && kustomize edit set image operator=$(IMAGE_REF)
+	controller-gen crd paths="$(GO_PACKAGES)" output:crd:artifacts:config=deploy/operator/
 
-	controller-gen $(CRD_OPTIONS) paths="$(GO_PACKAGES)" output:crd:dir=config/operator/crd/bases \
-	rbac:roleName=manager-role output:rbac:artifacts:config=config/operator/rbac \
-	webhook output:webhook:artifacts:config=config/operator/webhook
+	kubectl kustomize deploy/operator > examples/common/operator.yaml
```

Comment:
I was hoping we'd make a split so the deploy folder is about deploying the operator and manager and then examples would be related to different ScyllaCluster objects